students/AndyKwok/lesson02/Series.py

```diff
@@ -0,0 +1,38 @@
+# Description: Program that prints out Fibonacci or Lucas series depending on user input
+# Author: Andy Kwok
+# Last Updated: 07/01/2018
+# ChangeLog: 
+#			v1.0 - Initialization
+
+
+def	Fib_series(n):
```

Comment:
function names should follow snake_case convention -> `fib_series(n)`

also, looks like accidental extraneous space.